plugins/chat/lib/chat/user_notifications_extension.rb

```diff
@@ -6,137 +6,163 @@ def chat_summary(user, _ = nil)
       guardian = Guardian.new(user)
       return unless guardian.can_chat?
 
-      @messages =
-        Chat::Message
-          .joins(:user, :chat_channel)
-          .where.not(user: user)
-          .where("chat_messages.created_at > ?", 1.week.ago)
-          .joins("LEFT OUTER JOIN chat_mentions cm ON cm.chat_message_id = chat_messages.id")
-          .joins("LEFT OUTER JOIN chat_mention_notifications cmn ON cmn.chat_mention_id = cm.id")
-          .joins("LEFT OUTER JOIN notifications n ON cmn.notification_id = n.id")
-          .joins(
-            "INNER JOIN user_chat_channel_memberships uccm ON uccm.chat_channel_id = chat_channels.id",
-          )
-          .where(<<~SQL, user_id: user.id)
-          uccm.user_id = :user_id AND
-          (uccm.last_read_message_id IS NULL OR chat_messages.id > uccm.last_read_message_id) AND
-          (uccm.last_unread_mention_when_emailed_id IS NULL OR chat_messages.id > uccm.last_unread_mention_when_emailed_id) AND
-          (
-            (n.user_id = :user_id AND cmn.notification_id IS NOT NULL AND uccm.following IS true AND chat_channels.chatable_type = 'Category') OR
-            (chat_channels.chatable_type = 'DirectMessage')
-          )
-        SQL
-          .to_a
+      # TODO: handle muted & silenced users ?
```

Comment:
That wasn't handled before, and I didn't want to change too much of behaviour in this already too big PR.